sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -248,7 +248,7 @@ export interface DeletedCertificate extends KeyVaultCertificateWithPolicy {
 export type DeleteIssuerOptions = coreHttp.OperationOptions;
 
 // @public
-export type DeletionRecoveryLevel = 'Purgeable' | 'Recoverable+Purgeable' | 'Recoverable' | 'Recoverable+ProtectedSubscription' | 'CustomizedRecoverable+Purgeable' | 'CustomizedRecoverable' | 'CustomizedRecoverable+ProtectedSubscription';
+export type DeletionRecoveryLevel = string;
```

Comment:
Please let's make this a set of string literals. One can end with `| string` if necessary, though I would rather avoid having public API changes as part of this PR if possible.

Reply:
@sadasant A detailed discussion of this is available in the thread https://github.com/Azure/azure-sdk-for-js/pull/12821#discussion_r542616576. We have decided to move forward with this code.